dev-requirements.txt

```diff
@@ -1,4 +1,8 @@
 setuptools>=40.8.0
 vulture>=1.0
 bandit>=1.5.1
-pydocstyle>=3.0.0
\ No newline at end of file
+pydocstyle>=3.0.0
+codecov>=2.0.15
```

Comment:
@AHReccese Please remove `codecov` from here.
This package is deprecated and it has caused many problems in the past